buildbot/dependency.conf

```diff
@@ -32,7 +31,7 @@ ocloc_ver_win=27.20.100.9168
 cpu_driver_lin=2021.12.6.0.19
 cpu_driver_win=2021.12.6.0.19
 gpu_driver_lin=21.32.20609
-gpu_driver_win=30.0.100.9684
+gpu_driver_win=30.0.100.9805
```

Comment:
What is the difference between `gpu_driver_win` and `ocl_gpu_rt_ver_win`? Can they be different?
If no, can we remove the duplicate?

Reply:
One is for searching path on share folder, Another is for testing whether the driver version is correct. Their values were different before, although they are the same now. If we can ensure that they will be the same in the future, we can delete one .

Reply:
Another option to consider is to re-route the value to another variable:
<search_path>="use_the_name_from_driver_vesion"

This way we can still configure both values separately if needed and maintain a single version even if it's used in multiple places.

Anyway, it can be addressed separately. Please, make sure that Jenkins/Summary tests do not regress from this driver update.

Reply:
If we should delete one, please crate a Jira for me. It not only needs to change the configuration file, but also needs to change the CI source code. It needs some effort.